src/modules/caseNotes/components/EnrollmentCaseNotes.tsx

```diff
@@ -52,7 +57,6 @@ export const CASE_NOTE_COLUMNS = {
   NoteContentPreview: {
     key: 'content-preview',
     header: 'Note Content Preview',
-    maxWidth: '450px',
```

Comment:
Removed since this wasn't respected (maxWidth is not a valid property on ColumnDef and GenericTable doesn't do anything with it). There is a maxWidth applied to the Box below.